stackoverflow-dark.user.css

```diff
@@ -560,6 +560,22 @@ regexp("^https?:\/\/((?!area51)\w+\.)*?stackexchange\.com.*") {
     background-color: #242424;
     color: #eee;
   }
+  .top-bar .-secondary .-item .-link .indicator-badge,
+  .top-bar .-secondary .-link._danger-indicator::after {
+    border-color: black;
```

Comment:
Just go with straight forward colors for consistency pls.

Reply:
You mean using `#000` instead of a keyword?

Reply:
Yes.

Reply:
Fixed on both counts